src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -106,10 +117,21 @@ public bool IsSharedBindingConfigurationDetected
     public bool IsExportButtonEnabled => !ProgressReporter.IsOperationInProgress && IsCurrentProjectBound;
     public string ConnectionSelectionCaptionText => Connections.Any() ? UiResources.SelectConnectionToBindDescription : UiResources.NoConnectionExistsLabel;
 
+     public ManageBindingViewModel(
+         IConnectedModeServices connectedModeServices, 
+         ISolutionInfoProvider solutionInfoProvider,
+         IProgressReporterViewModel progressReporterViewModel)
+    {
+        this.connectedModeServices = connectedModeServices;
+        this.solutionInfoProvider = solutionInfoProvider;
+        ProgressReporter = progressReporterViewModel;
+    }
+    
     public async Task InitializeDataAsync()
     {
         var validationParams = new TaskToPerformParams<AdapterResponse>(LoadDataAsync, UiResources.LoadingConnectionsText, UiResources.LoadingConnectionsFailedText){AfterProgressUpdated = OnProgressUpdated};
         await ProgressReporter.ExecuteTaskWithProgressAsync(validationParams);
+        await DisplayBindStatusAsync();
```

Comment:
It would be more user-friendly to call the DisplayBindStatusAsync also by using the ProgressReporter, because we do an operation that might take a long (communicating with SlCore).This will show the progress in the UI